detection_rules/etc/version.lock.json

```diff
@@ -7270,25 +7284,25 @@
       }
     },
     "rule_name": "InstallUtil Process Making Network Connections",
-    "sha256": "3207a4319e151615e88e76119b8a5ab20e791feacf991e51958908a091c77775",
+    "sha256": "4574b9d8e17ff1d9a21c9cb08b31aa5a9499bcf303aa2f1fa5cef47684023dea",
     "type": "eql",
-    "version": 101
+    "version": 102
   },
   "a1329140-8de3-4445-9f87-908fb6d824f4": {
     "min_stack_version": "8.3",
     "previous": {
       "7.16": {
         "max_allowable_version": 99,
         "rule_name": "File Deletion via Shred",
-        "sha256": "f593f43ce7a9f78b7f49de94fbed61766e76d7721abd4ccc86f7b6f4f8edcb4f",
+        "sha256": "17e1166f1b8127f46a21c291885ad5397ddfec70435ffe0dd21873b74f3afe3c",
```

Comment:
Forked rule changed when all rules are min-stacked at 8.3 and thus changes would not backport to this branch.